crates/wkg/src/main.rs

```diff
@@ -269,6 +276,11 @@ enum Format {
 
 impl GetArgs {
     pub async fn run(self) -> anyhow::Result<()> {
+        ensure!(
+            !(self.overwrite && self.check),
```

Comment:
Can you add [`conflicts_with`](https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with) to the flags so this can be caught by the CLI?

Reply:
Thanks, I didn't remember that was possible with clap.